packages/trader/src/AppV2/Components/RiskManagementItem/risk-management-item.tsx

```diff
@@ -0,0 +1,175 @@
+                    </span>
+                    {!is_deal_cancellation &&
+                        (is_accumulator ? (
+                            <Text size='sm'>
+                                {finalValue} {currency}
+                            </Text>
+                        ) : (
+                            <ToggleSwitch
+                                disabled={isDealCancellation}
+                                checked={toggle}
+                                onChange={handleToggleSwitch}
+                            />
+                        ))}
+                    {is_valid_to_cancel && is_deal_cancellation && <DealCancellationRemainingTime />}
+                </div>
+                {!is_accumulator && toggle && (
+                    <TextField
+                        variant='fill'
+                        inputSize='md'
+                        disabled={isSheetOpen}
+                        textAlignment='center'
+                        value={`${finalValue.toFixed(2)} ${currency}`}
+                        onClick={() => {
+                            clearContractUpdateConfigValues();
+                            setStepperValue(finalValue);
+                            setIsSheetOpen(true);
+                        }}
+                        onFocus={() => setIsSheetOpen(true)}
+                    />
+                )}
+                <ActionSheet.Root
+                    expandable
+                    isOpen={isSheetOpen}
+                    position='left'
+                    onClose={() => {
+                        setIsEnabled(false);
+                        setIsSheetOpen(false);
+                    }}
+                >
+                    <ActionSheet.Portal>
+                        <ActionSheet.Header title={label} />
+                        <ActionSheet.Content>
+                            {isSheetOpen && (
+                                <TextFieldWithSteppers
+                                    variant='fill'
+                                    inputSize='md'
+                                    textAlignment='center'
+                                    status={errorMessage ? 'error' : 'neutral'}
+                                    name={type}
+                                    unit={currency}
+                                    value={Math.abs(stepperValue)}
+                                    onChange={onChange}
+                                    unitPlacement='right'
+                                    decimals={0}
+                                    message={errorMessage}
+                                />
+                            )}
+                        </ActionSheet.Content>
+                        <ActionSheet.Footer
+                            isPrimaryButtonDisabled={errorMessage !== ''}
+                            shouldCloseOnPrimaryButtonClick
+                            primaryAction={{
+                                content: localize('Save'),
```

Comment:
after latest quill updates, `content` is accepting `React.Node`, so we can pass `<Localize />`